### PR TITLE
Disable mime in setup

### PR DESCRIPTION
**Pull request by pengyin-shan**
Changes proposed in this PR:
- A update of README.md and CONTRIBUTION.md for 3.0 release
- Comment out the `rails mime_types:load` in `bin/setup` because it won't be part of 3.0 release

@lagoan the wiki belongs to its own repository so this PR doesn't include it (I will continue to work on wiki and update under #issue109). This PR contains the README update, Contribution guide update and `setup` file update.

### CODE REVIEW

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -140,13 +139,13 @@ The project team will evaluate each PR as time permits and communicate with the
  
 Once we receive your PR, at lest one member of the core development team will review the code and provide you with feedback through GitHub PR review feature. If any changes are requested, you should follow the process above to commit your additional changes, rebase again, and push your changes back up to Github. You do not need to close the PR and open another if you are working with the same branch. Note that if you rebase again you will need to force the push: `git push -f origin [my-branch]`
 
-### Acceptence of your PR
+### Acceptance of your PR
 
-Once your code has been approved a member of the core development team will merge it into the development branchand include it in an upcoming release. 
+Once your code has been approved a member of the core development team will merge it into the development branch and nd include it in an upcoming release.
```
Comment (lagoan) [inline .github/CONTRIBUTING.md:144]: Just a small typo here, the sentence has an extra `nd`

File: README.md [hunk 2]
```diff
@@ -1,64 +1,93 @@
-## DMP Roadmap
+<h1>DMP Assistant</h1>
```
Comment (lagoan) [inline README.md:1]: I think we should keep the `##` way of defining headers in markdown

File: README.md [hunk 3]
```diff
@@ -1,64 +1,93 @@
...
+> Developed by **the Digital Research Alliance of Canada (the Alliance)** in collaboration with host institution University of Alberta, DMP Assistant repo is a **fork** of the current state of the main <a href="https://github.com/DMPRoadmap/roadmap">DMP Roadmap codebase</a>, which is management and developed jointly provided by the Digital Curation Centre (DCC) and the University of California Curation Center (UC3).
 
-The tool has four main functions:
 
-1. To help create and maintain different versions of Data Management Plans;
-2. To provide useful guidance on data management issues and how to meet research funders' requirements;
-3. To export attractive and useful plans in a variety of formats;
-4. To allow collaborative work when creating Data Management Plans.
+---
+- [Latest Update](#overview)
```
Comment (lagoan) [inline README.md:14]: I think this should be 

```
[Latest Update](#latest-update)
```

**Review by lagoan — CHANGES_REQUESTED** [review]
All looks good! 

I just noted some minor suggestions.

**Comment by pengyin-shan** [comment]
> All looks good!
> 
> I just noted some minor suggestions.

Thank you! I made changes based on your view

**Review by lagoan — APPROVED** [review]
LGTM!

**Pull request closed without merging**